### ISSUE TITLE
server.listen is an asynchronous

### ISSUE THREAD

**Issue by shiryz** [FACN3/Auto-Completer#41]
since `listen` is asynchronous, the `Server is listening on PORT 4000` would log either way, whether or not the listen actually worked, the right way to do it is to add a callback to listen and log from there.
Also if you add the whole URI (http://localhost:9000) then in some shells you can click the link directly to open the app :+1:

**Comment by MynahMarie** [comment]
Good point!  About adding the whole URI, it doesn't work on Heroku if I do that... So thats why I changed it to a partial url. Will try and fix the callback though :)
